# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add first_weekday, most come from OpenOffice through CLDR
# DP:    This patch applies the following conventions:
# DP:       1=Sunday...7=Saturday
# DP:       The week keyword is removed from all locales.
# DP:    Normally GTK apps should work with these settings, as well as
# DP:    other apps.
# DP: Related bugs: #343885 347323
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

Index: localedata/locales/aa_DJ
===================================================================
--- localedata/locales/aa_DJ.orig
+++ localedata/locales/aa_DJ
@@ -273,5 +273,7 @@
 date_fmt    "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/aa_ER
===================================================================
--- localedata/locales/aa_ER.orig
+++ localedata/locales/aa_ER
@@ -223,5 +223,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/aa_ER@saaho
===================================================================
--- localedata/locales/aa_ER@saaho.orig
+++ localedata/locales/aa_ER@saaho
@@ -207,5 +207,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/aa_ET
===================================================================
--- localedata/locales/aa_ET.orig
+++ localedata/locales/aa_ET
@@ -226,5 +226,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/af_ZA
===================================================================
--- localedata/locales/af_ZA.orig
+++ localedata/locales/af_ZA
@@ -164,6 +164,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/am_ET
===================================================================
--- localedata/locales/am_ET.orig
+++ localedata/locales/am_ET
@@ -1564,5 +1564,7 @@
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/ar_AE
===================================================================
--- localedata/locales/ar_AE.orig
+++ localedata/locales/ar_AE
@@ -141,6 +141,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_BH
===================================================================
--- localedata/locales/ar_BH.orig
+++ localedata/locales/ar_BH
@@ -145,6 +145,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_DZ
===================================================================
--- localedata/locales/ar_DZ.orig
+++ localedata/locales/ar_DZ
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_EG
===================================================================
--- localedata/locales/ar_EG.orig
+++ localedata/locales/ar_EG
@@ -148,6 +148,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_IN
===================================================================
--- localedata/locales/ar_IN.orig
+++ localedata/locales/ar_IN
@@ -127,6 +127,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: copied from other *_IN locales
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/ar_IQ
===================================================================
--- localedata/locales/ar_IQ.orig
+++ localedata/locales/ar_IQ
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_JO
===================================================================
--- localedata/locales/ar_JO.orig
+++ localedata/locales/ar_JO
@@ -162,6 +162,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_KW
===================================================================
--- localedata/locales/ar_KW.orig
+++ localedata/locales/ar_KW
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_LB
===================================================================
--- localedata/locales/ar_LB.orig
+++ localedata/locales/ar_LB
@@ -161,6 +161,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_LY
===================================================================
--- localedata/locales/ar_LY.orig
+++ localedata/locales/ar_LY
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_MA
===================================================================
--- localedata/locales/ar_MA.orig
+++ localedata/locales/ar_MA
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_OM
===================================================================
--- localedata/locales/ar_OM.orig
+++ localedata/locales/ar_OM
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_QA
===================================================================
--- localedata/locales/ar_QA.orig
+++ localedata/locales/ar_QA
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_SA
===================================================================
--- localedata/locales/ar_SA.orig
+++ localedata/locales/ar_SA
@@ -132,6 +132,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_SD
===================================================================
--- localedata/locales/ar_SD.orig
+++ localedata/locales/ar_SD
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_SY
===================================================================
--- localedata/locales/ar_SY.orig
+++ localedata/locales/ar_SY
@@ -161,6 +161,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 5
 END LC_TIME
 
 
Index: localedata/locales/ar_TN
===================================================================
--- localedata/locales/ar_TN.orig
+++ localedata/locales/ar_TN
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/ar_YE
===================================================================
--- localedata/locales/ar_YE.orig
+++ localedata/locales/ar_YE
@@ -144,6 +144,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: localedata/locales/az_AZ
===================================================================
--- localedata/locales/az_AZ.orig
+++ localedata/locales/az_AZ
@@ -274,6 +274,8 @@
 t_fmt     "<U0025><U0054>"
 am_pm     "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/be_BY
===================================================================
--- localedata/locales/be_BY.orig
+++ localedata/locales/be_BY
@@ -145,6 +145,9 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  Copied from be_BY@alternative
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/be_BY@latin
===================================================================
--- localedata/locales/be_BY@latin.orig
+++ localedata/locales/be_BY@latin
@@ -102,6 +102,9 @@
 date_fmt        "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  Copied from be_BY@alternative
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_COLLATE
Index: localedata/locales/bn_IN
===================================================================
--- localedata/locales/bn_IN.orig
+++ localedata/locales/bn_IN
@@ -145,6 +145,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/br_FR
===================================================================
--- localedata/locales/br_FR.orig
+++ localedata/locales/br_FR
@@ -141,6 +141,8 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "<U0020>";"<U0020>"
 t_fmt_ampm "<U0025><U0049><U0065><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/bs_BA
===================================================================
--- localedata/locales/bs_BA.orig
+++ localedata/locales/bs_BA
@@ -208,6 +208,8 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/byn_ER
===================================================================
--- localedata/locales/byn_ER.orig
+++ localedata/locales/byn_ER
@@ -218,5 +218,7 @@
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12A3><U12F5>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/ca_ES
===================================================================
--- localedata/locales/ca_ES.orig
+++ localedata/locales/ca_ES
@@ -138,6 +138,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/cs_CZ
===================================================================
--- localedata/locales/cs_CZ.orig
+++ localedata/locales/cs_CZ
@@ -361,7 +361,7 @@
 %		"<U004E><U006F><U0076>";/
 %		"<U0044><U0065><U0063>"
 
-week		7;19971201;4
+% week		7;19971201;4
 
 d_t_fmt		"<U0025><U0061><U00A0><U0025><U0065><U002E><U00A0><U0025><U0042><U00A0><U0025><U0059><U002C><U00A0><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U00A0><U0025><U005A>"
 
@@ -376,6 +376,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 
 END LC_TIME
 
Index: localedata/locales/csb_PL
===================================================================
--- localedata/locales/csb_PL.orig
+++ localedata/locales/csb_PL
@@ -191,7 +191,7 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
-week    7;19971201;4
+% week    7;19971201;4
 first_weekday 2
 first_workday 2
 END LC_TIME
Index: localedata/locales/cy_GB
===================================================================
--- localedata/locales/cy_GB.orig
+++ localedata/locales/cy_GB
@@ -253,6 +253,8 @@
 date_fmt    "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/da_DK
===================================================================
--- localedata/locales/da_DK.orig
+++ localedata/locales/da_DK
@@ -214,6 +214,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/de_AT
===================================================================
--- localedata/locales/de_AT.orig
+++ localedata/locales/de_AT
@@ -121,6 +121,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/de_BE
===================================================================
--- localedata/locales/de_BE.orig
+++ localedata/locales/de_BE
@@ -119,6 +119,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/de_CH
===================================================================
--- localedata/locales/de_CH.orig
+++ localedata/locales/de_CH
@@ -121,6 +121,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/de_DE
===================================================================
--- localedata/locales/de_DE.orig
+++ localedata/locales/de_DE
@@ -118,7 +118,7 @@
 	"<U0044><U006F><U006E><U006E><U0065><U0072><U0073><U0074><U0061><U0067>";/
 	"<U0046><U0072><U0065><U0069><U0074><U0061><U0067>";/
 	"<U0053><U0061><U006D><U0073><U0074><U0061><U0067>"
-week    7;19971201;4
+% week    7;19971201;4
 abmon	"<U004A><U0061><U006E>";"<U0046><U0065><U0062>";/
 	"<U004D><U00E4><U0072>";"<U0041><U0070><U0072>";/
 	"<U004D><U0061><U0069>";"<U004A><U0075><U006E>";/
@@ -153,6 +153,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U002D><U0064><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
 <U0020><U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/de_LU
===================================================================
--- localedata/locales/de_LU.orig
+++ localedata/locales/de_LU
@@ -119,6 +119,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/dz_BT
===================================================================
--- localedata/locales/dz_BT.orig
+++ localedata/locales/dz_BT
@@ -1544,7 +1544,8 @@
 %I<U0F40><U0F66><U0F62><U0F0B><U0F58>/
 %M<U0F40><U0F66><U0F62><U0F0B><U0F46>%S %p"
 
-
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/el_GR
===================================================================
--- localedata/locales/el_GR.orig
+++ localedata/locales/el_GR
@@ -139,6 +139,8 @@
 %
 % Appropriate date representation (date(1))   "%a %d %b %Y %r %Z"
 date_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0072><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/en_AU
===================================================================
--- localedata/locales/en_AU.orig
+++ localedata/locales/en_AU
@@ -117,6 +117,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_CA
===================================================================
--- localedata/locales/en_CA.orig
+++ localedata/locales/en_CA
@@ -132,6 +132,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_DK
===================================================================
--- localedata/locales/en_DK.orig
+++ localedata/locales/en_DK
@@ -106,6 +106,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_GB
===================================================================
--- localedata/locales/en_GB.orig
+++ localedata/locales/en_GB
@@ -121,6 +121,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_HK
===================================================================
--- localedata/locales/en_HK.orig
+++ localedata/locales/en_HK
@@ -133,6 +133,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0070><U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/en_IE
===================================================================
--- localedata/locales/en_IE.orig
+++ localedata/locales/en_IE
@@ -115,6 +115,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_IN
===================================================================
--- localedata/locales/en_IN.orig
+++ localedata/locales/en_IN
@@ -132,6 +132,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/en_NZ
===================================================================
--- localedata/locales/en_NZ.orig
+++ localedata/locales/en_NZ
@@ -117,6 +117,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_PH
===================================================================
--- localedata/locales/en_PH.orig
+++ localedata/locales/en_PH
@@ -133,6 +133,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/en_SG
===================================================================
--- localedata/locales/en_SG.orig
+++ localedata/locales/en_SG
@@ -131,6 +131,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/en_US
===================================================================
--- localedata/locales/en_US.orig
+++ localedata/locales/en_US
@@ -81,7 +81,7 @@
 	"<U0046><U0072><U0069><U0064><U0061><U0079>";/
 	"<U0053><U0061><U0074><U0075><U0072><U0064><U0061><U0079>"
 
-week    7;19971130;7
+% week    7;19971130;7
 abmon	"<U004A><U0061><U006E>";"<U0046><U0065><U0062>";/
 	"<U004D><U0061><U0072>";"<U0041><U0070><U0072>";/
 	"<U004D><U0061><U0079>";"<U004A><U0075><U006E>";/
@@ -125,6 +125,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/en_ZA
===================================================================
--- localedata/locales/en_ZA.orig
+++ localedata/locales/en_ZA
@@ -202,6 +202,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/es_AR
===================================================================
--- localedata/locales/es_AR.orig
+++ localedata/locales/es_AR
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_BO
===================================================================
--- localedata/locales/es_BO.orig
+++ localedata/locales/es_BO
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_CL
===================================================================
--- localedata/locales/es_CL.orig
+++ localedata/locales/es_CL
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_CO
===================================================================
--- localedata/locales/es_CO.orig
+++ localedata/locales/es_CO
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_CR
===================================================================
--- localedata/locales/es_CR.orig
+++ localedata/locales/es_CR
@@ -113,6 +113,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_DO
===================================================================
--- localedata/locales/es_DO.orig
+++ localedata/locales/es_DO
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_EC
===================================================================
--- localedata/locales/es_EC.orig
+++ localedata/locales/es_EC
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_GT
===================================================================
--- localedata/locales/es_GT.orig
+++ localedata/locales/es_GT
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_HN
===================================================================
--- localedata/locales/es_HN.orig
+++ localedata/locales/es_HN
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_MX
===================================================================
--- localedata/locales/es_MX.orig
+++ localedata/locales/es_MX
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_NI
===================================================================
--- localedata/locales/es_NI.orig
+++ localedata/locales/es_NI
@@ -114,6 +114,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_PA
===================================================================
--- localedata/locales/es_PA.orig
+++ localedata/locales/es_PA
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_PE
===================================================================
--- localedata/locales/es_PE.orig
+++ localedata/locales/es_PE
@@ -123,6 +123,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_PR
===================================================================
--- localedata/locales/es_PR.orig
+++ localedata/locales/es_PR
@@ -113,6 +113,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_PY
===================================================================
--- localedata/locales/es_PY.orig
+++ localedata/locales/es_PY
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_SV
===================================================================
--- localedata/locales/es_SV.orig
+++ localedata/locales/es_SV
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_US
===================================================================
--- localedata/locales/es_US.orig
+++ localedata/locales/es_US
@@ -174,6 +174,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_UY
===================================================================
--- localedata/locales/es_UY.orig
+++ localedata/locales/es_UY
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/es_VE
===================================================================
--- localedata/locales/es_VE.orig
+++ localedata/locales/es_VE
@@ -122,6 +122,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/et_EE
===================================================================
--- localedata/locales/et_EE.orig
+++ localedata/locales/et_EE
@@ -177,6 +177,8 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/eu_ES
===================================================================
--- localedata/locales/eu_ES.orig
+++ localedata/locales/eu_ES
@@ -131,6 +131,7 @@
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/fr_BE
===================================================================
--- localedata/locales/fr_BE.orig
+++ localedata/locales/fr_BE
@@ -122,6 +122,7 @@
 <U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
 <U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/fr_CA
===================================================================
--- localedata/locales/fr_CA.orig
+++ localedata/locales/fr_CA
@@ -123,6 +123,7 @@
 <U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
 <U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
+first_weekday 1
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/fr_CH
===================================================================
--- localedata/locales/fr_CH.orig
+++ localedata/locales/fr_CH
@@ -107,6 +107,7 @@
 <U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
 <U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/fr_FR
===================================================================
--- localedata/locales/fr_FR.orig
+++ localedata/locales/fr_FR
@@ -131,6 +131,7 @@
 <U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
 <U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/fr_LU
===================================================================
--- localedata/locales/fr_LU.orig
+++ localedata/locales/fr_LU
@@ -122,6 +122,7 @@
 <U0025><U0042><U0020><U0025><U0059><U002C><U0020>/
 <U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0028><U0055><U0054><U0043><U0025><U007A><U0029>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/ga_IE
===================================================================
--- localedata/locales/ga_IE.orig
+++ localedata/locales/ga_IE
@@ -117,6 +117,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/gd_GB
===================================================================
--- localedata/locales/gd_GB.orig
+++ localedata/locales/gd_GB
@@ -122,6 +122,8 @@
 t_fmt       "<U0025><U0054>"
 am_pm       "";""
 t_fmt_ampm  ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/gez_ER
===================================================================
--- localedata/locales/gez_ER.orig
+++ localedata/locales/gez_ER
@@ -215,5 +215,7 @@
 <U1361><U0025><U0072><U1361><U0025><U005A><U1361><U0025><U0059>/
 <U1361><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/gez_ET
===================================================================
--- localedata/locales/gez_ET.orig
+++ localedata/locales/gez_ET
@@ -216,5 +216,7 @@
 <U1361><U0025><U0072><U1361><U0025><U005A><U1361><U0025><U0059>/
 <U1361><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/gl_ES
===================================================================
--- localedata/locales/gl_ES.orig
+++ localedata/locales/gl_ES
@@ -121,6 +121,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/gu_IN
===================================================================
--- localedata/locales/gu_IN.orig
+++ localedata/locales/gu_IN
@@ -161,6 +161,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/gv_GB
===================================================================
--- localedata/locales/gv_GB.orig
+++ localedata/locales/gv_GB
@@ -131,6 +131,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/he_IL
===================================================================
--- localedata/locales/he_IL.orig
+++ localedata/locales/he_IL
@@ -123,6 +123,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/hi_IN
===================================================================
--- localedata/locales/hi_IN.orig
+++ localedata/locales/hi_IN
@@ -156,6 +156,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/hr_HR
===================================================================
--- localedata/locales/hr_HR.orig
+++ localedata/locales/hr_HR
@@ -220,6 +220,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/hsb_DE
===================================================================
--- localedata/locales/hsb_DE.orig
+++ localedata/locales/hsb_DE
@@ -220,6 +220,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/hu_HU
===================================================================
--- localedata/locales/hu_HU.orig
+++ localedata/locales/hu_HU
@@ -553,7 +553,9 @@
 t_fmt_ampm "<U0025><U0048><U002E><U0025><U004D><U002E><U0025><U0053>"
 date_fmt "<U0025><U0059><U002E><U0020><U0025><U0062><U002E><U0020><U0025><U0065><U002E><U002C>/
 <U0020><U0025><U0041><U002C><U0020><U0025><U0048><U002E><U0025><U004D><U002E><U0025><U0053><U0020><U0025><U005A>"
-week     7;19971201;4
+% week     7;19971201;4
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/hy_AM
===================================================================
--- localedata/locales/hy_AM.orig
+++ localedata/locales/hy_AM
@@ -317,6 +317,8 @@
 t_fmt       "<U0025><U0072>"
 am_pm       "";""
 t_fmt_ampm  ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/id_ID
===================================================================
--- localedata/locales/id_ID.orig
+++ localedata/locales/id_ID
@@ -120,6 +120,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/is_IS
===================================================================
--- localedata/locales/is_IS.orig
+++ localedata/locales/is_IS
@@ -2223,6 +2223,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in Java
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/it_CH
===================================================================
--- localedata/locales/it_CH.orig
+++ localedata/locales/it_CH
@@ -98,6 +98,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/it_IT
===================================================================
--- localedata/locales/it_IT.orig
+++ localedata/locales/it_IT
@@ -124,6 +124,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/ja_JP
===================================================================
--- localedata/locales/ja_JP.orig
+++ localedata/locales/ja_JP
@@ -15006,6 +15006,8 @@
 		"<U4E5D><U5341><U56DB>";"<U4E5D><U5341><U4E94>";/
 		"<U4E5D><U5341><U516D>";"<U4E5D><U5341><U4E03>";/
 		"<U4E5D><U5341><U516B>";"<U4E5D><U5341><U4E5D>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/kk_KZ
===================================================================
--- localedata/locales/kk_KZ.orig
+++ localedata/locales/kk_KZ
@@ -232,6 +232,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/km_KH
===================================================================
--- localedata/locales/km_KH.orig
+++ localedata/locales/km_KH
@@ -1841,6 +1841,8 @@
 % ICT-7ICT-7
 %timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
 
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/kn_IN
===================================================================
--- localedata/locales/kn_IN.orig
+++ localedata/locales/kn_IN
@@ -165,6 +165,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/ko_KR
===================================================================
--- localedata/locales/ko_KR.orig
+++ localedata/locales/ko_KR
@@ -6189,6 +6189,8 @@
 <U0020><U0028><U0025><U0061><U0029>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
 <U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 1
 
 END LC_TIME
 
Index: localedata/locales/ku_TR
===================================================================
--- localedata/locales/ku_TR.orig
+++ localedata/locales/ku_TR
@@ -149,6 +149,8 @@
 t_fmt_ampm  ""
 % format: %A %d %B %Y
 date_fmt  "<U0025><U0041><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/kw_GB
===================================================================
--- localedata/locales/kw_GB.orig
+++ localedata/locales/kw_GB
@@ -123,6 +123,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/lo_LA
===================================================================
--- localedata/locales/lo_LA.orig
+++ localedata/locales/lo_LA
@@ -718,6 +718,8 @@
 <U003A><U0025><U0053><U0020><U0025><U005A>"
 % ICT-7ICT-7
 timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/lt_LT
===================================================================
--- localedata/locales/lt_LT.orig
+++ localedata/locales/lt_LT
@@ -137,6 +137,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/lv_LV
===================================================================
--- localedata/locales/lv_LV.orig
+++ localedata/locales/lv_LV
@@ -2183,6 +2183,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/mk_MK
===================================================================
--- localedata/locales/mk_MK.orig
+++ localedata/locales/mk_MK
@@ -85,9 +85,9 @@
 date_fmt "<U0025><U0061><U002C><U0020><U0025><U0064><U0020><U0025><U0062>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
-week    7;19971201;4
-first_weekday 1 % Monday
-first_workday 1 % Monday
+% week    7;19971201;4
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MONETARY
Index: localedata/locales/ml_IN
===================================================================
--- localedata/locales/ml_IN.orig
+++ localedata/locales/ml_IN
@@ -145,6 +145,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm     "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 %
 LC_MESSAGES
Index: localedata/locales/mn_MN
===================================================================
--- localedata/locales/mn_MN.orig
+++ localedata/locales/mn_MN
@@ -220,6 +220,8 @@
 % "%Z %Y ony %B %e, %a %H:%M:%S"
 date_fmt  "<U0025><U005A><U0020><U0025><U0059><U0020><U043E><U043D><U044B><U0020><U0025><U0042><U0020>/
 <U0025><U0065><U002C><U0020><U0025><U0061><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/mr_IN
===================================================================
--- localedata/locales/mr_IN.orig
+++ localedata/locales/mr_IN
@@ -153,6 +153,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/ms_MY
===================================================================
--- localedata/locales/ms_MY.orig
+++ localedata/locales/ms_MY
@@ -139,6 +139,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: localedata/locales/nb_NO
===================================================================
--- localedata/locales/nb_NO.orig
+++ localedata/locales/nb_NO
@@ -195,6 +195,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/ne_NP
===================================================================
--- localedata/locales/ne_NP.orig
+++ localedata/locales/ne_NP
@@ -153,6 +153,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/nl_BE
===================================================================
--- localedata/locales/nl_BE.orig
+++ localedata/locales/nl_BE
@@ -104,6 +104,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/nl_NL
===================================================================
--- localedata/locales/nl_NL.orig
+++ localedata/locales/nl_NL
@@ -123,6 +123,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/nn_NO
===================================================================
--- localedata/locales/nn_NO.orig
+++ localedata/locales/nn_NO
@@ -136,6 +136,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: localedata/locales/nr_ZA
===================================================================
--- localedata/locales/nr_ZA.orig
+++ localedata/locales/nr_ZA
@@ -162,6 +162,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/nso_ZA
===================================================================
--- localedata/locales/nso_ZA.orig
+++ localedata/locales/nso_ZA
@@ -157,6 +157,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
  
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/oc_FR
===================================================================
--- localedata/locales/oc_FR.orig
+++ localedata/locales/oc_FR
@@ -123,6 +123,8 @@
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_NAME
Index: localedata/locales/om_ET
===================================================================
--- localedata/locales/om_ET.orig
+++ localedata/locales/om_ET
@@ -233,5 +233,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/pa_IN
===================================================================
--- localedata/locales/pa_IN.orig
+++ localedata/locales/pa_IN
@@ -149,6 +149,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/pt_BR
===================================================================
--- localedata/locales/pt_BR.orig
+++ localedata/locales/pt_BR
@@ -121,6 +121,7 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+first_weekday 1
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/pt_PT
===================================================================
--- localedata/locales/pt_PT.orig
+++ localedata/locales/pt_PT
@@ -124,6 +124,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/rw_RW
===================================================================
--- localedata/locales/rw_RW.orig
+++ localedata/locales/rw_RW
@@ -109,6 +109,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/se_NO
===================================================================
--- localedata/locales/se_NO.orig
+++ localedata/locales/se_NO
@@ -277,6 +277,8 @@
 <U0020><U0025><U005A>"
 
 % %a %b-%e %H:%M:%S %Z %Y sn jan-31 21:15:11 CET 2001
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/sid_ET
===================================================================
--- localedata/locales/sid_ET.orig
+++ localedata/locales/sid_ET
@@ -222,5 +222,7 @@
 %
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/sk_SK
===================================================================
--- localedata/locales/sk_SK.orig
+++ localedata/locales/sk_SK
@@ -183,7 +183,10 @@
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 
-week		7;19971201;4
+% week		7;19971201;4
+
+%  FIXME: found in CLDR
+first_weekday 2
 
 END LC_TIME
 
Index: localedata/locales/sl_SI
===================================================================
--- localedata/locales/sl_SI.orig
+++ localedata/locales/sl_SI
@@ -157,6 +157,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/so_DJ
===================================================================
--- localedata/locales/so_DJ.orig
+++ localedata/locales/so_DJ
@@ -222,5 +222,7 @@
 date_fmt    "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/so_ET
===================================================================
--- localedata/locales/so_ET.orig
+++ localedata/locales/so_ET
@@ -219,5 +219,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/so_KE
===================================================================
--- localedata/locales/so_KE.orig
+++ localedata/locales/so_KE
@@ -218,5 +218,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/so_SO
===================================================================
--- localedata/locales/so_SO.orig
+++ localedata/locales/so_SO
@@ -278,5 +278,7 @@
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/sq_AL
===================================================================
--- localedata/locales/sq_AL.orig
+++ localedata/locales/sq_AL
@@ -290,6 +290,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: localedata/locales/sr_CS
===================================================================
--- localedata/locales/sr_CS.orig
+++ localedata/locales/sr_CS
@@ -304,7 +304,7 @@
 date_fmt  "<U0025><U0061><U002C><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
 <U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
-week    7;19971201;4
+% week    7;19971201;4
 first_weekday 2
 first_workday 2
 END LC_TIME
Index: localedata/locales/ss_ZA
===================================================================
--- localedata/locales/ss_ZA.orig
+++ localedata/locales/ss_ZA
@@ -167,6 +167,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/st_ZA
===================================================================
--- localedata/locales/st_ZA.orig
+++ localedata/locales/st_ZA
@@ -164,6 +164,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/sv_FI
===================================================================
--- localedata/locales/sv_FI.orig
+++ localedata/locales/sv_FI
@@ -105,6 +105,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/ta_IN
===================================================================
--- localedata/locales/ta_IN.orig
+++ localedata/locales/ta_IN
@@ -230,6 +230,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/te_IN
===================================================================
--- localedata/locales/te_IN.orig
+++ localedata/locales/te_IN
@@ -139,6 +139,8 @@
 % %p%I.%M.%S %Z
 t_fmt_ampm  "<U0025><U0070><U0025><U0049><U002E><U0025><U004D><U002E><U0025><U0053><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/th_TH
===================================================================
--- localedata/locales/th_TH.orig
+++ localedata/locales/th_TH
@@ -913,6 +913,8 @@
 <U003A><U0025><U0053><U0020><U0025><U005A>"
 % ICT-7ICT-7
 timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/ti_ER
===================================================================
--- localedata/locales/ti_ER.orig
+++ localedata/locales/ti_ER
@@ -270,5 +270,7 @@
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/ti_ET
===================================================================
--- localedata/locales/ti_ET.orig
+++ localedata/locales/ti_ET
@@ -1072,5 +1072,7 @@
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/tig_ER
===================================================================
--- localedata/locales/tig_ER.orig
+++ localedata/locales/tig_ER
@@ -218,5 +218,7 @@
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/tn_ZA
===================================================================
--- localedata/locales/tn_ZA.orig
+++ localedata/locales/tn_ZA
@@ -169,6 +169,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/tr_TR
===================================================================
--- localedata/locales/tr_TR.orig
+++ localedata/locales/tr_TR
@@ -1540,6 +1540,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/ts_ZA
===================================================================
--- localedata/locales/ts_ZA.orig
+++ localedata/locales/ts_ZA
@@ -164,6 +164,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/tt_RU
===================================================================
--- localedata/locales/tt_RU.orig
+++ localedata/locales/tt_RU
@@ -297,6 +297,8 @@
 t_fmt     "<U0025><U0054>"
 am_pm     "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: localedata/locales/uz_UZ
===================================================================
--- localedata/locales/uz_UZ.orig
+++ localedata/locales/uz_UZ
@@ -260,6 +260,8 @@
 am_pm       "";""
 t_fmt_ampm  ""
 date_fmt    "<U0025><U0064><U0020><U0025><U0042><U002C><U0020><U0025><U0059><U0020><U0079><U0069><U006C><U002C><U0020><U0025><U0041>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/uz_UZ@cyrillic
===================================================================
--- localedata/locales/uz_UZ@cyrillic.orig
+++ localedata/locales/uz_UZ@cyrillic
@@ -263,6 +263,8 @@
 am_pm       "";""
 t_fmt_ampm  ""
 date_fmt    "<U0025><U0064><U0020><U0025><U0042><U002C><U0020><U0025><U0059><U0020><U0439><U0438><U043B><U002C><U0020><U0025><U0041>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/ve_ZA
===================================================================
--- localedata/locales/ve_ZA.orig
+++ localedata/locales/ve_ZA
@@ -166,6 +166,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/vi_VN
===================================================================
--- localedata/locales/vi_VN.orig
+++ localedata/locales/vi_VN
@@ -296,6 +296,8 @@
 am_pm    "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U0020><U0025><U0070>"
 date_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 % yes: Va^ng / Da.
Index: localedata/locales/wa_BE
===================================================================
--- localedata/locales/wa_BE.orig
+++ localedata/locales/wa_BE
@@ -163,7 +163,8 @@
 am_pm   "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
 % prum djo del samwinne (1=londi)
-first_weekday 1
+% FIXME  No, 2=lundi
+first_weekday 2
 END LC_TIME
 
 LC_NAME
Index: localedata/locales/wal_ET
===================================================================
--- localedata/locales/wal_ET.orig
+++ localedata/locales/wal_ET
@@ -232,5 +232,7 @@
 <U0020><U130B><U120B><U1233>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U130D><U002F><U120B>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: localedata/locales/xh_ZA
===================================================================
--- localedata/locales/xh_ZA.orig
+++ localedata/locales/xh_ZA
@@ -164,6 +164,8 @@
 % time-zone name,
 % year with century as a decimal number,e.g. 2001.
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/yi_US
===================================================================
--- localedata/locales/yi_US.orig
+++ localedata/locales/yi_US
@@ -186,6 +186,8 @@
 am_pm   "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0050>"
 cal_direction	3
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/zh_CN
===================================================================
--- localedata/locales/zh_CN.orig
+++ localedata/locales/zh_CN
@@ -135,6 +135,8 @@
 date_fmt       "<U0025><U0059><U5E74><U0020><U0025><U006D><U6708><U0020>/
 <U0025><U0064><U65E5><U0020><U0025><U0041><U0020><U0025><U0048><U003A>/
 <U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/zh_HK
===================================================================
--- localedata/locales/zh_HK.orig
+++ localedata/locales/zh_HK
@@ -141,6 +141,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: localedata/locales/zh_SG
===================================================================
--- localedata/locales/zh_SG.orig
+++ localedata/locales/zh_SG
@@ -130,6 +130,8 @@
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  ""
 %
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: localedata/locales/zh_TW
===================================================================
--- localedata/locales/zh_TW.orig
+++ localedata/locales/zh_TW
@@ -128,6 +128,9 @@
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 
+%  FIXME: found in CLDR
+first_weekday 1
+
 END LC_TIME
 
 LC_MESSAGES
Index: localedata/locales/zu_ZA
===================================================================
--- localedata/locales/zu_ZA.orig
+++ localedata/locales/zu_ZA
@@ -170,6 +170,8 @@
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
